cirq/vis/heatmap_test.py

```diff
@@ -243,3 +243,27 @@ def test_colorbar(ax):
     # TODO: Make this is a more thorough test, e.g., we should test that the
     # position, size and pad arguments are respected.
     # Github issue: https://github.com/quantumlib/Cirq/issues/2969
+
+
+def test_interheatmap():
+    value_map = {
+        (grid_qubit.GridQubit(3, 2), grid_qubit.GridQubit(4, 2)): 0.004619111460557768,
+        (grid_qubit.GridQubit(4, 1), grid_qubit.GridQubit(4, 2)): 0.0076079162393482835,
+    }
+    value_map1 = {((3, 2), (4, 2)): 0.004619111460557768, ((4, 1), (4, 2)): 0.0076079162393482835}
+    test_heatmap = heatmap.InterHeatmap(value_map)
+    test_heatmap1 = heatmap.InterHeatmap(value_map1)
+    test_url_map = {(3.5, 2): 'http://google.com/1', (4, 1.5): 'http://google.com/2'}
+    test_heatmap.unset_url_map().set_url_map(test_url_map)
+    test_heatmap.unset_colorbar().set_colorbar()
+
+    colormap_name = 'Greys'
+    test_heatmap.set_colormap(0.001, 0.01, colormap_name)
+
+    annot_map = {(3.5, 2): '.3e', (4, 1.5): '.2f'}
+    test_heatmap.unset_annotation()
+    test_heatmap.set_annotation_map(annot_map)
+    test_heatmap.set_annotation_format('.3e')
+
+    test_heatmap.plot()
+    test_heatmap1.plot()
```

Comment:
this test is not testing anything

Reply:
Yeah that's a problems. What way do you suggest to test when result is a graph?